Patches/Dragon's Descent/DamageDefs/Patch_Damage_Dragon.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- DamageDefs. first time patching this. -->
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<!-- ==== Check for the mod. ==== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Dragon's Descent</modName>
+			</li>
+
+			<!-- ==== Add DefModExtension ==== -->
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/DamageDef[defName="Toxic_Bite" or defName="Rend" or defName="Skull_Bash" or defName="Frost_Bite" or defName="Pyro_Bite"]</xpath>
```

Comment:
Unless the `Toxic_Bite` from this mod does something special, I'd recommend substituting it for the vanilla `ToxicBite`. which CE has already patched. If you do that, you can remove it from needing to be changed in your patch.

Reply:
I'll change all mentions of `Toxic_Bite` in the patch to `ToxicBite`, as the hediff `Toxic_Bite` is a copy of `ToxicBuildup`.